continuous-integration/linux/platform-dependent/ubuntu-18.04/setup.sh

```diff
@@ -22,19 +23,43 @@ update-alternatives --install /usr/bin/python python /usr/bin/python3 0 || exit
 update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 0 || exit $?
 
 function install_opencl_cpu_runtime {
```

Comment:
Should move the part that installs the intel driver to a common place instead of duplicating it?

Reply:
Moved